docs/source/changes.rst

```diff
@@ -94,8 +135,13 @@
 * cleaned up the code
 * Fixed overflow bug in Minute-Count+
 
-### Version 0.2 2019-04-26 Juergen Boehringer
+V0.2 2019-04-26 jb
+------------------
+
 * added 1 Minute RS232 (USB) logging mode
 
-### Version 0.1 2019-03-25 Juergen Boehringer
+V0.1 2019-03-25 jb
```

Comment:
I needed to shorten that a bit, so it looks prettier when rendered in the left sidebar on the generated html site.